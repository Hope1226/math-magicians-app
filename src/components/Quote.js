import React from 'react';
import './stylesheets/quote.css';

const Quote = () => (
  <div className="quoteContainer">
    <p>
      Mathematics is not about numbers, equations, computations,
      or  or algorithms: it is about understanding - William Poul Thurston
    </p>
  </div>
);

export default Quote;
