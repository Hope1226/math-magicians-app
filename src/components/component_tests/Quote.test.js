import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

describe('Loads the Quote page', () => {
  it('and matches the snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
