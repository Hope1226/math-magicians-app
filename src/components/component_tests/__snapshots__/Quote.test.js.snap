// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loads the Quote page and matches the snapshot 1`] = `[Function]`;
